assets/javascripts/test_result.js

```diff
@@ -861,6 +921,19 @@ function renderInvestigationTab(response) {
   tabPanelElement.dataset.initialized = true;
 }
 
+function toggleSign(elem) {
+  if (elem.className === 'collapsed' && elem.innerHTML === '+ ') {
+    elem.innerHTML = '- ';
+  } else {
+    elem.innerHTML = '+ ';
```

Comment:
https://www.javascripttutorial.net/javascript-ternary-operator/ plz

Reply:
*this change is ready for review*